.github/releasers/releaser_gui_centos.sh

```
@@ -0,0 +1,57 @@
+	--license "${LICENSE}" \
+	--maintainer "${MAINTAINER}" \
+	--description "${DESC}" \
+	--url "${URL}" \
+	--category "${CATEGORY}" \
+	--rpm-os linux \
+	"${BUILD_DIR}/aerium-daemon"=/usr/local/bin/aerium-daemon \
+	"${BUILD_DIR}/aerium-shell"=/usr/local/bin/aerium-shell \
+	"${BUILD_DIR}/aerium-wallet"=/usr/local/bin/aerium-wallet \
+	"${BUILD_DIR}/aerium-gui"=/usr/local/bin/aerium-gui \
+	"${ROOT_DIR}/.github/releasers/linux/aerium-gui.desktop"=/usr/share/applications/aerium-gui.desktop \
+	${ROOT_DIR}/cmd/gtk/assets/images/logo.png"=/usr/share/icons/hicolor/256x256/apps/aerium.png
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)